config/environments/development.rb

```diff
@@ -73,5 +73,5 @@
 
   config.action_mailer.delivery_method = :letter_opener_web
   config.action_mailer.perform_caching = true
-  config.action_mailer.default_url_options = { :host => ENV['PRODUCTION_HOST'] }
+  config.action_mailer.default_url_options = { :host => ENV['LOCAL_HOST'] }
```

Comment:
it should be just host

Reply:
Should I just rename it or something more?